dapp/src/components/AccountListener.js

```diff
@@ -344,6 +353,7 @@ const AccountListener = (props) => {
     } else {
       balancesQuery.refetch()
       allowancesQuery.refetch()
+      historyQuery.refetch()
```

Comment:
I would probably rather refetch history only when `account` or `chainId` changes in `useEffect` below the one I am commenting on. To reduce the amount of fetches.